apps/private-law/prl-cron-fm5-reminder/demo.yaml

```diff
@@ -7,54 +7,7 @@ spec:
   releaseName: prl-cron-fm5-reminder
   values:
     job:
-      keyVaults:
-        prl:
-          secrets:
-            - name: prl-cos-idam-client-secret
-              alias: IDAM_CLIENT_SECRET
-            - name: microservicekey-prl-cos-api
-              alias: IDAM_S2S_AUTH_TOTP_SECRET
-            - name: system-update-user-username
-              alias: SYSTEM_UPDATE_USER_USERNAME
-            - name: system-update-user-password
-              alias: PRL_SYSTEM_UPDATE_PASSWORD
-            - name: notification-client-api-key
-              alias: UK_GOV_NOTIFY_API_KEY
-            - name: test-genericpassword
-              alias: PRL_GENERIC_PASSWORD
-            - name: launchDarkly-sdk-key
-              alias: LAUNCH_DARKLY_SDK_KEY
-            - name: send-grid-api-key
-              alias: SEND_GRID_API_KEY
-            - name: rpa-notify-to-email
-              alias: RPA_NOTIFY_TO_EMAIL
-            - name: rpa-notify-from-email
-              alias: RPA_NOTIFY_FROM_EMAIL
-            - name: AppInsightsInstrumentationKey
-              alias: APP_INSIGHTS_INSTRUMENTATION_KEY
-            - name: southampton-court-email-address
-              alias: SOUTHAMPTON_COURT_EMAIL_ADDRESS
-            - name: idam-solicitor-username
-              alias: IDAM_SOLICITOR_USERNAME
-            - name: idam-solicitor-password
-              alias: IDAM_SOLICITOR_PASSWORD
-            - name: filtered-court-ids
-              alias: COURT_IDS_TO_FILTER
-            - name: da-filtered-court-ids
-              alias: DA_COURT_IDS_TO_FILTER
-            - name: ca-default-court-epimms-id
-              alias: CA_DEFAULT_COURT_EPIMMS_ID
-            - name: postcode-lookup-token
-              alias: POST_CODE_LOOKUP_KEY
-            - name: prl-rd-api-system-username
-              alias: PRL_IDAM_REFDATA_USERNAME
-            - name: prl-rd-api-system-user-password
-              alias: PRL_IDAM_REFDATA_PASSWORD
-            - name: welsh-court-mapping
-              alias: WELSH_COURT_MAPPING
-            - name: app-insights-connection-string
-              alias: app-insights-connection-string
-      image: hmctspublic.azurecr.io/prl/cos:pr-2544-e809114-20240912102254 #{"$imagepolicy": "flux-system:demo-prl-cron-fm5-reminder"}
+      image: hmctspublic.azurecr.io/prl/cos:prod-835c14b-20240911080538 #{"$imagepolicy": "flux-system:prl-cos"}
```

Comment:
prl-cos is it valid?

Reply:
Yes @AlokDatta, it will refer prl-cos & pull latest prod image